javascript-data-model/order-history.js

```diff
@@ -0,0 +1,103 @@
+var orderHistory = [
+
+  {
+    orderDetails: {
+      orderNumber: '114-3941689-8772232',
+      shipTo: 'JS Masher',
+      total: 34.00,
+      orderPlaced: 'August 4, 2020',
+      deliveryStatus: ['Yes', 'No'],
+      deliveryDate: ['August 8, 2020', 'In Transit'],
+      returnWindow: 'Closed on Sep 7, 2020'
+    },
+
+    productDetails: {
+      productName: 'Javascript for impatient programmers',
+      price: 31.55,
+      author: ['Yes', 'No'],
+      authorName: ['Dr. Axel Rauschmayer', 'No Author']
+    }
+
+  },
+
+  {
+    orderDetails: {
+      orderNumber: '113-9984268-128025',
+      shipTo: 'JS Masher',
+      total: 44.53,
+      orderPlaced: 'July 19, 2020',
+      deliveryStatus: ['Yes', 'No'],
+      deliveryDate: ['Jul 20, 2020', 'In Transit'],
+      returnWindow: 'Closed on Aug 19, 2020'
+    },
+
+    productDetails: {
+      productName: 'The Timeless Way of Building',
+      price: 41.33,
+      author: ['Yes', 'No'],
+      authorName: ['Christopher Alexander', 'No Author']
+    }
+
+  },
+
+  {
+    orderDetails: {
+      orderNumber: '114-2875557-9059409',
+      shipTo: 'JS Masher',
+      total: 17.22,
+      orderPlaced: 'July  4, 2020',
+      deliveryStatus: ['Yes', 'No'],
+      deliveryDate: ['July 7, 2020', 'In Transit'],
+      returnWindow: 'Closed on August 5, 2020'
+    },
+
+    productDetails: {
+      productName: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+      price: 15.98,
+      author: ['Yes', 'No'],
+      authorName: [null, 'No Author']
+    }
+
+  },
+
+  {
+    orderDetails: {
+      orderNumber: '113-2883177-2648248',
+      shipTo: 'JS Masher',
+      total: 138.93,
+      orderPlaced: 'July 3, 2020',
+      deliveryStatus: ['Yes', 'No'],
+      deliveryDate: ['July 5, 2020', 'In Transit'],
+      returnWindow: 'Closed on August 4, 2020'
+    },
+
+    productDetails: {
+      productName: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      price: 94.95,
+      author: ['Yes', 'No'],
+      authorName: [null, 'No Author']
+    }
+
+  },
+
+  {
+    orderDetails: {
+      orderNumber: '113-2883177-2648248',
+      shipTo: 'JS Masher',
+      total: 138.93,
+      orderPlaced: 'July 3, 2020',
+      deliveryStatus: ['Yes', 'No'],
+      deliveryDate: ['July 5, 2020', 'In Transit'],
+      returnWindow: 'Closed on August 4, 2020'
+    },
+
+    productDetails: {
+      productName: 'The Art of Sql',
+      price: 33.99,
+      author: ['Yes', 'No'],
+      authorName: ['Stephanie, Faroult', 'No Author']
+    }
+
+  }
```

Comment:
By having these as separate objects, they look like separate orders. You should have one object to represent the order that is capable of holding anywhere from 1-infinity number of items.